gt-scheduler/website
# Emily/208 minor UI enhancements

### Summary

Resolves #208 


Adds a popup modal when the user attempts to remove a user they invited to share their schedule
Adds text to inform a user that they have to invite someone when they haven't invited anyone

### Checklist

- [ ] Popup is displayed with the email of the user's about to be removed.
- [ ] When no friends have been invited to the current schedule, a text is displayed in the background as a holder.


### How to Test

Open the "Share Schedule" modal from the "export" tab in the action bar
Click on the "x" to remove a user and the remove modal should pop up

Remove all users from the list of invited users, and the prompting text should be visible
